### ISSUE TITLE
'No space left on device' error when create new conda env

### ISSUE THREAD

**Issue by YanZhangADS** [Azure/batch-scoring-for-dl-models#17]
I got 'No space left on device' error after executing "conda env create -f environment.yml". 
1. How do I know how much space left in my device in order to accommodate this new conda env?

2. We should include only the must-have packages in the environment.yml file.

**Comment by fboylu** [comment]
I agree that the environment file is very long, probably created by using conda env export. is there a way to start with clean environment and manually create the env file?
